app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt

```diff
@@ -73,6 +76,13 @@ class TransactionListFragment : SafeOverviewBaseFragment<FragmentTransactionList
             }
         }
 
+        binding.transactions.addOnScrollListener(object : RecyclerView.OnScrollListener() {
+            override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
+                val firstVisibleItem = (binding.transactions.layoutManager as LinearLayoutManager).findFirstVisibleItemPosition()
+                reload = firstVisibleItem <= 1
```

Comment:
Just curious. Can this be negative? What would that mean if it is negative?

Reply:
it would mean recycler view has no children and in this case, scrolling event won't be triggered at all